src/Livewire/Settings/UserEdit.php

```diff
@@ -121,22 +122,26 @@ public function show(?int $id = null): void
 
     public function save(): void
     {
-        if (
-            in_array(
-                resolve_static(Role::class, 'findByName', ['name' => 'Super Admin'])->id,
-                $this->user['roles']
-            )
-            && ! auth()->user()->hasRole('Super Admin')
-        ) {
-            return;
+        try {
+            if (
+                in_array(
+                    resolve_static(Role::class, 'findByName', ['name' => 'Super Admin'])->id,
+                    $this->user['roles']
+                )
+                && ! auth()->user()->hasRole('Super Admin')
+            ) {
+                return;
+            }
+        } catch (RoleDoesNotExist) {
         }
 
         $action = ($this->user['id'] ?? false) ? UpdateUser::class : CreateUser::class;
 
         try {
             $user = $action::make($this->user)
                 ->checkPermission()
-                ->setRules($this->getRules())
+                ->setRulesFromRulesets()
+                ->mergeRules($this->getRules())
```

Comment:
can the getRules() method be removed? As this method is not very helpful.
Also mergeRules with ['password' => ['confirmed']] does override the other password validation rules, and only 'confirmed' remains.